docs/pages/enroll-resources/application-access/cloud-apis/aws-console-without-agent.mdx

```diff
@@ -0,0 +1,304 @@
+1. Copy the Profile ARN <Var name="Profile ARN" />
+
+### Create a Teleport AWS IAM Roles Anywhere integration
+Now that the required AWS resources are created, you can create a Teleport AWS IAM Roles Anywhere integration.
+
+1. Write the following contents to a file called `roles-anywhere-integration.yaml`:
+  ```yaml
+  kind: integration
+  sub_kind: aws-ra
+  version: v1
+  metadata:
+    name: <Var name="Integration Name" />
+  spec:
+    aws_ra:
+      trust_anchor_arn: <Var name="Trust Anchor ARN" />
+      profile_sync_config:
+        enabled: true
+        profile_arn: <Var name="Profile ARN" />
+        role_arn: <Var name="Role ARN" />
+  ```
+
+1. Create the integration with the following command:
+  ```code
+  $ tctl create -f roles-anywhere-integration.yaml
+  ```
+
+Teleport will now start syncing AWS IAM Roles Anywhere Profiles as AWS application resources, every 5 minutes.
+
+You can get the sync summary by running the following command:
+  ```code
+  $ tctl get integration/<Var name="Integration Name" />
+  ...
+  status:
+    aws_ra:
+      last_profile_sync:
+        end_time: "2025-07-23T15:46:27.475629Z"
+        error_message: ""
+        start_time: "2025-07-23T15:46:26.334536Z"
+        status: SUCCESS
+        synced_profiles: 4
+  ```
+
+## Step 2/4. Create an AWS IAM Roles Anywhere Profile and assign IAM Roles
+
+In this step, you will create an example Profile and Role so that you can test the integration.
+
+If you are already leveraging AWS IAM Roles Anywhere Profiles, you can skip this step.
+
+1. Navigate to [Create Role](https://console.aws.amazon.com/iam/home?#/roles/create) and create a new IAM Role called `ExampleReadOnlyAccess`.
+Trust policy must include the Roles Anywhere service principal:
+  ```json
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Principal": {
+                  "Service": "rolesanywhere.amazonaws.com"
+              },
+              "Action": [
+                  "sts:AssumeRole",
+                  "sts:SetSourceIdentity",
+                  "sts:TagSession"
+              ]
+          }
+      ]
+  }
+  ```
+
+Add the AWS-managed `ReadOnlyAccess` policy to the role.
+Copy the Role ARN <Var name="Read Only Role ARN" />.
+
+1. Now, navigate to [Create a Profile](https://console.aws.amazon.com/rolesanywhere/home/profiles/create) page and name it `ExampleReadOnlyAccess`.
+Add the role created in the last step (`ExampleReadOnlyAccess`) and create the Profile.
+
+Copy the Profile ARN <Var name="Read Only Profile ARN" />.
+
+1. You should now see the `ExampleReadOnlyAccess` profile listed in your Teleport Web UI in the Resources page.
+
+## Step 3/4. Set up Access
+In this section, you'll create a Teleport Role which will grant access to the `ExampleReadOnlyAccess` Profile and associated IAM Role.
+
+1. Create a file named `example-read-only-access-role.yaml` with the following contents:
```

Comment:
Can we instead show how to do this in the web UI with role editor? Even if it's yaml, it's easier to create it in the web UI.

Reply:
I thought about this, but given this is a Docs guide, I assumed we prefer a more CLI oriented flow.
That's also why we use `tctl` to create the integration instead of using the UI.

I see 3 options here:
- keep it as is (tctl only)
- swap this with UI instructions
- show both flows and use a Tab element so the user can switch between them (see the Trust Anchor CA set up step above - `curl` vs `tctl`

Reply:
Added tabs for UI and CLI 👍